### ISSUE TITLE
🐛[BUG]System cant log in

### ISSUE THREAD

**Issue by Slayerfaith** [HorizonFFXI/HorizonXI-Issues#1093]
BUG/SYSTEM
Stuck in Bastok Metalworks, as soon as I log in the game crashes and goes right back to the desktop

https://user-images.githubusercontent.com/125772128/219949893-a75da252-4508-44c2-8654-8ac8d695d212.mp4

**Comment by gtarrantiii** [comment]
Start simple, disable all your addons. As it is happening right as you load in, the addons/plugins are still loading and might be crashing your game. Disable anything that didn't come with HXI and try again.
Start adding items back on one at a time until you find the issue. 
If you didnt add anything else, disable addons in the HXI launcher. 
Try again from there.

**Comment by Slayerfaith** [comment]
I turned them all off. If it even matters I was using allmaps checker timers and tparty, but the result is the same

**Comment by gtarrantiii** [comment]
Go into your Horizon install folder, there will be a logs folder, post the most recent file.
Path will be something like:
...\HorizonXI\Game\logs

**Comment by Slayerfaith** [comment]
[02_21_2023 07.08.12.txt](https://github.com/HorizonFFXI/HorizonXI-Issues/files/10798454/02_21_2023.07.08.12.txt)

**Comment by gtarrantiii** [comment]
Thank you for that, I looked at the log and it appears to be failing while doing DirectX binding of your input devices, or at least that is where the logs are stopping. 
Have you been noticing any input issues or changed hardware recently? 
Do you recall a time that it was working where we can compare a before and after log? 
A reinstall might address it if you are not getting any other error information to go on.

**Comment by Slayerfaith** [comment]
i've barely even lagged until saturday afternoon, and the only thing i feel is out of place is ever since i tried downloading discord something isn't right. as soon as i hit my desktop it starts booting up or asking what app i'd like to use to open, even after i've tried to uninstall and delete from my hard drive. it even shows up on the app store as not on my pc yet still acts this way. then again maybe totally unrelated

**Comment by Slayerfaith** [comment]
[02_15_2023 17.48.21.txt](https://github.com/HorizonFFXI/HorizonXI-Issues/files/10798632/02_15_2023.17.48.21.txt)

**Comment by gtarrantiii** [comment]
That log looks more like normal execution. You might try running a virus scan and see if anything comes up. 
Otherwise it looks like you were running and able to log in, if you were not sent to jail or banned for any reason then you can try a reinstall. 
Try discord in your web browser to visit the Horizon discord server and check the launcher support postings or post your symptoms there, hopefully a member of the staff can get back with you further, link to this issue. https://discord.com/channels/933423693848260678/1048400974676496425

This doesnt jump out to me as an apparent bug at this time, although the devs behind the launcher might have more info to go on with the documented behaviour above.

**Comment by Slayerfaith** [comment]
did the quick and full scan, came up clean. a reinstall won't lose my character data correct? may just do that

**Comment by gtarrantiii** [comment]
Yea, char info is held server side, the only thing you could lose is window layouts, addon configs, local macros.. stuff like that.
To save some of those, copy the  ...\HorizonXI\Game\config folder out for reference later if you need them to recover those settings.

**Comment by Slayerfaith** [comment]
thank you very much

**Comment by Slayerfaith** [comment]
As it turns out, the problem was a corrupted file which is odd as I'd been logged in for sometime then boom kicked off. I tried removing everything but the install file only to get error 0x80070570. I'm not good with tech but I ended up doing a reinstall which seems to have worked, but I'm leaving this here to suggest to anyone else who may experience this might try repairing their hard drive. I'm not positive but maybe if I'd known to try that first I could have avoided the reinstall altogether...? Just leaving for whomever it may help. Cheers
